tests/test_parsers.py

```diff
@@ -93,6 +93,29 @@ class Flake8ParserTestCase(ParserTestCaseMixin, unittest.TestCase):
     }
 
 
+class BanditParserTestCase(ParserTestCaseMixin, unittest.TestCase):
```

Comment:
Since the parser is called BanditJSONParser, perhaps this should be called BanditJSONParserTestCase too?